React/Views/RCTScrollView.m

```diff
@@ -725,6 +725,28 @@ - (void)scrollViewDidEndZooming:(UIScrollView *)scrollView withView:(UIView *)vi
   RCT_FORWARD_SCROLL_EVENT(scrollViewDidEndZooming:scrollView withView:view atScale:scale);
 }
 
+- (void)scrollViewDidEndDecelerating:(UIScrollView *)scrollView
+{
+  // Fire a final scroll event
+  _allowNextScrollNoMatterWhat = YES;
+  [self scrollViewDidScroll:scrollView];
```

Comment:
Are you suggesting that in JavaScript, developers must subscribe to `onScroll` for most of the scroll position information and then `onMomentumScrollEnd` to find out what the resting scroll position is? If so, this design of the `onScroll` event seems broken to me. If developers care about scroll position, it's likely that they'll sign up for the `onScroll` event but not realize that they also need to sign up for the `onMomentumScrollEnd` event resulting in a bug in their app. Because of this, it seems to me that it would be preferable to fire an `onScroll` event as we are doing now.

What do you think?